tests/test_context_cloning.c

```diff
@@ -21,16 +21,16 @@ int main() {
 		return EXIT_FAILURE;
 	}
 
-	// 0xd1 or 209 is space (0x20 or \u0120 in tokenizer)
-	const unsigned char * prompt = "hello\xd1world";
+	const unsigned char prompt[12] = "hello world";
 
 	rwkv_eval(ctx, prompt[0], NULL, state, logits);
 
-	for (const unsigned char * token = prompt + 1; *token != 0; token++) {
-		rwkv_eval(ctx, *token, state, state, logits);
+	for (int i = 1; prompt[i] != 0; i++) {
```

Comment:
should this be just `prompt[i]`, not `prompt[i] != 0`?

Reply:
I think this would be a valid simplification, but explicit comparison feels more clear to me.

Reply:
It's a common practice in C when reading strings, for what it's worth, but either way is fine I think